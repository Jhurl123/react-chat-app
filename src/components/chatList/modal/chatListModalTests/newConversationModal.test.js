import React from 'react';
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Modal from '@material-ui/core/Modal';
import NewConversationModal from '../NewConversationModal'

describe('NewConversationModal Component', () => {

  const open = false
  const toggleModal = jest.fn()

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<NewConversationModal open={open} toggleModal={toggleModal} />)
      .toJSON();
      expect(tree).toMatchSnapshot();  
  });

  it('renders the UI with text content', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NewConversationModal open={open} toggleModal={toggleModal} />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
  
})