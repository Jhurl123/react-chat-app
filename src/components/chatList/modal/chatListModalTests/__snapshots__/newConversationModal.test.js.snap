// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewConversationModal Component renders the UI as expected 1`] = `null`;
